src/main/java/eu/europeana/metis/sandbox/service/workflow/TransformationServiceImpl.java

```diff
@@ -41,12 +51,44 @@ public RecordInfo transform(Record record) {
     return new RecordInfo(Record.from(record, recordTransformed));
   }
 
+  @Override
+  public RecordInfo transformToEdmExternal(Record record) {
+    requireNonNull(record, "Record must not be null");
+    String xsltToEdmExternal = datasetRepository.getOne(Integer.valueOf(
+        record.getDatasetId())).getXsltTransformerEdmExternal();
+
+    return new RecordInfo(Record.from(record, transformToEdmExternal(record.getDatasetId(),
+        record.getDatasetName(),
+        new ByteArrayInputStream(xsltToEdmExternal.getBytes(StandardCharsets.UTF_8)),
+        record.getCountry(), record.getLanguage(), record.getContent())));
+  }
+
+  @Override
+  public byte[] transformToEdmExternal(String datasetId, String datasetName, InputStream xsltToEdmExternal,
```

Comment:
Does this method care what we are transforming to?
The input stream can be anything, so this can have a more general name as well as its parameters.
Also the parameters of datasetName, country and language are specific to the transformation to internal and here they are not required.

Reply:
That's a good point, I named these as they are since the main intention is for transformation to EDM external. I could even name the method `transform` to `transformToEdmInternal` to be more specific and name this one `transform`, so basically switching up their names, what do you think?

Reply:
Not sure about renaming the other one, since that is a generic one as well as we can simply change the xslt provided to the transformer to something else. On the other hand it's quite specific with the parameter and such so yes we could. I'll leave this up to you.

Reply:
I switched the method names